src/core/errorHandler.js

```diff
@@ -31,41 +36,57 @@ export class ErrorHandler {
 
 class ErrorHandlerComponent extends React.Component {
   static propTypes = {
+    autoRenderErrors: PropTypes.boolean,
     dispatch: PropTypes.func,
     error: PropTypes.object,
     errorHandlerId: PropTypes.string,
     WrappedComponent: PropTypes.object,
   }
 
+  static defaultProps = {
+    autoRenderErrors: true,
+  }
+
   render() {
     const {
+      autoRenderErrors,
       WrappedComponent,
       errorHandlerId,
       dispatch,
       error,
       ...props
     } = this.props;
-    const errorHandler = new ErrorHandler({ id: errorHandlerId, dispatch });
-    const wrappedOutput = (
-      <WrappedComponent errorHandler={errorHandler} {...props} />
-    );
 
+    let errorContent;
     if (error) {
+      errorContent = (
+        <ul className="ErrorHandler-list">
+          {error.messages.map((msg) => <li>{msg}</li>)}
+        </ul>
+      );
+    }
+
+    const errorHandler = new ErrorHandler({
+      errorContent,
+      id: errorHandlerId,
+      dispatch,
+    });
+    const allProps = { ...props, errorHandler };
+
+    if (error && autoRenderErrors) {
```

Comment:
Yes, you are correct. By passing `autoRenderErrors = false`, the component is responsible for rendering errors itself. However, if the component forgets to do so then that could be confusing. I will log a warning so that at least there's something in the console.

This feature actually came up in the original review but I wanted to wait until we absolutely needed it. Now we need it!